pkgs/by-name/en/encore/package.nix

```
@@ -0,0 +1,59 @@
+{ fetchFromGitHub
+, buildGoModule
+, callPackage
+, makeWrapper
+, lib
+, ...
+}:
+buildGoModule rec {
+  pname = "encore";
+  version = "1.39.0";
+
+  src = fetchFromGitHub {
+    name = "${pname}-source";
+    owner = "encoredev";
+    repo = pname;
```

Please avoid superfluous use of `pname` (for details see #277994)
```suggestion
    name = "encore-source";
    owner = "encoredev";
    repo = encore;
```